database/src/main/java/database/service/JsonDatabaseService.java

```diff
@@ -145,9 +165,30 @@ public boolean removeRecordFromTable(Class<? extends BaseEntity> entityClass, In
         return true;
     }
 
+    @Override
+    public boolean removeSpecificRecords(Class<? extends BaseEntity> entityClass, List<Integer> ids) {
+        if (ids == null || ids.isEmpty()) {
+            throw new IllegalArgumentException(IDS_LIST_NULL_OR_EMPTY);
```

Comment:
Please use custom exception which extends from `BadRequestException` to return 400 error code instead of 500.